[2022-08-27 22:06:08,362] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-25T00:00:00+00:00 [queued]>
[2022-08-27 22:06:08,374] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-25T00:00:00+00:00 [queued]>
[2022-08-27 22:06:08,375] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-27 22:06:08,375] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-27 22:06:08,376] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-27 22:06:08,394] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): link_dag> on 2022-08-25 00:00:00+00:00
[2022-08-27 22:06:08,403] {standard_task_runner.py:52} INFO - Started process 384 to run task
[2022-08-27 22:06:08,406] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'link_dag', 'scheduled__2022-08-25T00:00:00+00:00', '--job-id', '257', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpiujrlve1', '--error-file', '/tmp/tmp704yeoa6']
[2022-08-27 22:06:08,408] {standard_task_runner.py:80} INFO - Job 257: Subtask link_dag
[2022-08-27 22:06:08,474] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.link_dag scheduled__2022-08-25T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-27 22:06:08,550] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=link_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-25T00:00:00+00:00
[2022-08-27 22:06:52,919] {python.py:173} INFO - Done. Returned value was: None
[2022-08-27 22:06:52,929] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=link_dag, execution_date=20220825T000000, start_date=20220827T220608, end_date=20220827T220652
[2022-08-27 22:06:52,948] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-27 22:06:52,972] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
